tests/pkg/certs/certprovider/ext_cert_provider.go

```diff
@@ -51,7 +51,7 @@ func NewExternalCertFromConfig(t *testing.T, ctx context.Context, testConfig Ext
 		k8s.PrintJobLogs(t, ctx, k8sClient, testConfig.ExternalCertTestJobName, testConfig.ExternalClientCertTestSecretNamespace, testConfig.ExternalCertCronjobContainerName, false)
 
 		k8s.DeleteJob(t, ctx, k8sClient, testConfig.ExternalCertTestJobName, testConfig.ExternalClientCertTestSecretNamespace)
-		k8s.DeleteSecret(t, ctx, k8sClient, testConfig.ExternalClientCertTestSecretName, testConfig.ExternalClientCertTestSecretNamespace)
+		//k8s.DeleteSecret(t, ctx, k8sClient, testConfig.ExternalClientCertTestSecretName, testConfig.ExternalClientCertTestSecretNamespace)
```

Comment:
Hm, this commit was made by another colleague, and to be honest, I'm not sure if it's intentional or not. I would guess it's not because this secret should be deleted at the end and with the current implementation seems it's not deleted. I'll double-check and see what can be done here.